src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/WinformsDefaultFontUpdater.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Windows
+{
+    [ApplicableComponents(ProjectComponents.WinForms)]
+    public class WinformsDefaultFontUpdater : IUpdater<IProject>
+    {
+        private readonly WindowsUtilities _utilities = new();
+        private readonly ILogger<WinformsDefaultFontUpdater> _logger;
+
+        public string Id => typeof(WinformsDefaultFontUpdater).FullName;
+
+        public string Title => "Default Font API Alert";
+
+        public string Description => "Default font in Windows Forms has been changed from Microsoft Sans Serif to Seg Segoe UI, in order to change the default font use the API - Application.SetDefaultFont(Font font). For more details see here - https://devblogs.microsoft.com/dotnet/whats-new-in-windows-forms-in-net-6-0-preview-5/#application-wide-default-font";
+
+        public BuildBreakRisk Risk => BuildBreakRisk.Low;
+
+        public WinformsDefaultFontUpdater(ILogger<WinformsDefaultFontUpdater> logger)
+        {
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+
+        public Task<IUpdaterResult> ApplyAsync(IUpgradeContext context, ImmutableArray<IProject> inputs, CancellationToken token)
```

Comment:
Why does the apply not do anything? Is the `IsApplicableAsync` step doing something to the project?

Reply:
Its not, its only logging a warning. that is why the Apply is just calling the IsApplicable